packages/crayons-core/src/components/tabs/tabs.scss

```diff
@@ -1,10 +1,8 @@
 /**
   @prop --fw-tabs-width: width of the tab container.
   @prop --fw-tabs-height: height of the tab container.
-  @prop --fw-tabs-margin-l: left margin for the tab items
```

Comment:
these are going to be breaking changes!!. Please call out in the PR. For CO_UI , they had a need for having different left and right margin and that is why there are separate variables for each margin.

Reply:
Noted. Will retain these names.

Reply:
Fixed.